telegram/ext/_handler.py

```diff
@@ -72,6 +77,9 @@ async def callback(update: Update, context: CallbackContext)
             be awaited before processing the next handler in
             :meth:`telegram.ext.Application.process_update`. Defaults to :obj:`True`.
 
+            .. seealso:: `Concurrency <https://github.com/\
+                python-telegram-bot/python-telegram-bot/wiki/Concurrency>`_
+
```

Comment:
this should probably be added to all handlers :D

Reply:
Done.

BTW, I think there's something wrong in `Attributes` section. For example, in `conversationhandler.py`:

```
    Attributes:
        block (:obj:`bool`): Determines whether the callback will run in a blocking way.. Always
            :obj:`True` since conversation handlers handle any non-blocking callbacks internally.
```

This renders in HTML as:
```
Determines whether the callback will run in a blocking way..

    Type:

        [bool](https://docs.python.org/3/library/functions.html#bool)
```

Reply:
the second sentence is there in the online docs though 🤔 https://docs.python-telegram-bot.org/en/v20.0a4/telegram.ext.conversationhandler.html#telegram.ext.ConversationHandler.block

Reply:
Yes, I must have looked somewhere else